jetty-websocket/websocket-jetty-tests/src/test/java/org/eclipse/jetty/websocket/tests/JettyWebSocketFilterTest.java

```diff
@@ -45,18 +50,29 @@
     private WebSocketClient client;
     private ServletContextHandler contextHandler;
 
-    @BeforeEach
-    public void start() throws Exception
+    public void start(JettyWebSocketServletContainerInitializer.Configurator configurator) throws Exception
+    {
+        start(configurator, null);
+    }
+
+    public void start(ServletHolder servletHolder) throws Exception
+    {
+        start(null, servletHolder);
+    }
+
+    public void start(JettyWebSocketServletContainerInitializer.Configurator configurator, ServletHolder servletHolder) throws Exception
     {
         server = new Server();
         connector = new ServerConnector(server);
         server.addConnector(connector);
 
         contextHandler = new ServletContextHandler(ServletContextHandler.SESSIONS);
         contextHandler.setContextPath("/");
+        if (servletHolder != null)
+            contextHandler.addServlet(servletHolder, "/");
         server.setHandler(contextHandler);
 
-        JettyWebSocketServletContainerInitializer.configure(contextHandler, null);
+        JettyWebSocketServletContainerInitializer.configure(contextHandler, configurator);
```

Comment:
Can we put a check within `configure` on the start of the contextHandler passed?  If it is started then it should ISE.   No sure about if it is starting?

Reply:
Ah I just saw #5406, so ignore this one.